compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -252,6 +256,7 @@ impl<'cx, 'tcx> crate::MirBorrowckCtxt<'cx, 'tcx> {
         desc: &str,
         is_arg: bool,
     ) -> DiagnosticBuilder<'cx, ErrorGuaranteed> {
+        //FIXME: the trait bound `(): SessionDiagnostic<'_>` is not satisfied
```

Comment:
You'll need to share what you tried when you got this error for me to be able to help. :)